### ISSUE TITLE
Math symbol not readable

### ISSUE THREAD

**Issue by tonicebrian** [arXiv/html_feedback#3218]
### Description

 Math symbol not readable

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

03762e52-e404-4a87-bac1-96bb9c0ea6a2

### Paper URL

https://arxiv.org/html/2410.06011v1

### Browser

Chrome/132.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
@tonicebrian could you provide a little more detail about the readability problem you encountered here?

Here is a screenshot from Chrome I am examining (dark) and its comparison to PDF (light):

<img width="800" src="https://github.com/user-attachments/assets/1093fa8d-4412-4dbc-a180-a4cbddcfb5a6">

<img width="400" src="https://github.com/user-attachments/assets/20e215ff-2056-415d-8209-ab0e17a7f9bb">

**Comment by tonicebrian** [comment]
Hi @dginev ups sorry I think I was confused about the way the indicator symbol is rendered in my end. This is what I see in my Ubuntu + Chrome machine:

![Image](https://github.com/user-attachments/assets/051a8d74-c7c3-4a2b-919a-3c5309a605b8)

to me it looked like the `[]` caret symbol you get when something is missing, but on your screenshot it clearly looks like a capital I. Feel free to close since it seems like a Linux/Ubuntu issue.

**Comment by dginev** [comment]
Thank you, it is great to have your screenshot.

Let us keep this open a little longer, we may want to consider adjusting font size or font family, if there are clear paths to improve legibility.
